test/e2e/testsuites/specs.go

```diff
@@ -62,6 +68,8 @@ type VolumeDetails struct {
 	DataSource *DataSource
 	// Optional, used with specified StorageClass
 	StorageClass *storagev1.StorageClass
+	// Optional, used for sorting by label of AzVolumeAttachments
+	PersistentVolume *v1.PersistentVolume
```

Comment:
This is also defined at line 64.
https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/blob/7127d7a27befd2c3d9435544e59dd2d7e8bf1eaf/test/e2e/testsuites/specs.go#L64